packages/loader/container-loader/src/container.ts

```diff
@@ -1342,12 +1396,22 @@ export class Container extends EventEmitterWithErrorHandling<IContainerEvents> i
         return client;
     }
 
+    /**
+     * Returns true if connection is active, i.e. it's "write" connection and
+     * container runtime was notified about this connection (i.e. we are up-to-date and could send ops)
```

Comment:
might want to mention that you are in the quorum or you have received the Join op, which is the condition for be `Connected` in write mode.